src/default-xpring-client.ts

```diff
@@ -205,6 +271,43 @@ class DefaultXpringClient implements XpringClientDecorator {
     const getFeeRequest = this.networkClient.GetFeeRequest()
     return this.networkClient.getFee(getFeeRequest)
   }
+
+  private async getAccountData(address: string): Promise<AccountRoot> {
+    const account = this.networkClient.AccountAddress()
+    account.setAddress(address)
+
+    const request = this.networkClient.GetAccountInfoRequest()
+    request.setAccount(account)
+
+    const accountInfo = await this.networkClient.getAccountInfo(request)
+    if (!accountInfo) {
+      throw new Error(XpringClientErrorMessages.malformedResponse)
+    }
+
+    const accountData = accountInfo.getAccountData()
+    if (!accountData) {
+      throw new Error(XpringClientErrorMessages.malformedResponse)
+    }
+
+    return accountData
+  }
+
+  /**
+   * Convert a polymorphic numeric value into a BigInteger.
+   *
+   * @param value The value to convert.
+   * @returns A BigInteger representing the input value.
+   */
+  private static toBigInt(value: string | number | BigInteger): BigInteger {
+    if (typeof value === 'string') {
+      return bigInt(value)
+    }
+    if (typeof value === 'number') {
+      return bigInt(value)
+    }
+    // Value is already a BigInteger.
+    return value
```

Comment:
There should be no harm in returning `bigInt(someBigInteger)` so I think this entire function can be replaced with just `return bigInt(value)`

Reply:
This actually throws a bunch of typing errors (it wants a unary value, not a complex value type).

Reply:
Oh, this does work. Weirdly, VSCode complains but it compiles!